README.md

```diff
@@ -160,6 +165,29 @@ docker run -it \
 
 <!-- CONFIGURATION -->
 
+## Lavalink 配置
```

Comment:
Maybe add some instructions on how to run the script? Like a python command. Or, `docker run` to run the script inside the environment of the image.